modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/files/Close.java

```
@@ -55,10 +54,8 @@
     @Override
     public BValue[] execute(Context context) {
         BStruct struct = (BStruct) getRefArgument(context, 0);
-        BufferedInputStream is = (BufferedInputStream) struct.getNativeData("inStream");
-        BufferedOutputStream os = (BufferedOutputStream) struct.getNativeData("outStream");
-        closeQuietly(is);
-        closeQuietly(os);
+        SeekableByteChannel channel = (SeekableByteChannel) struct.getNativeData("channel");
+        closeQuietly(channel);
```

Why do we close quietly?  We need to throw an error here.